docs/conf.py

```diff
@@ -34,18 +34,18 @@
     "sphinx_design",
     "sphinx_copybutton",
     "autoapi.extension",
-    "_extension.gallery_directive",
     # For extension examples and demos
+    "myst_parser",
```

Comment:
I'm a little confused about why `myst_parser` wasn't needed before. Was it previously included as a dependency of `myst_nb`?